default/python/universe_generation/planets.py

```diff
@@ -4,27 +4,26 @@
 import util
 import universe_tables as tables
 
-
```

Comment:
Why remove this blank line? These two lines separate two major sections (imports and global constants), I like that to be a bit more visually distinct...